### DISCUSSION TITLE
Why info not change in demo even i already change on demo.yml?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1403] [Q&A] [answered]
Hi all,

First of all i'm using docker method. I'm try understand the attribute by change some value on demo.yml. But the display on dashboard still not chamge anything. For example value that i change was to "mangkuk-edge1.example.com". Appreciate any guide.

- snmp:
      name: mangkuk-edge1.example.com
      interfaces:
        10: "Transit: Telia"
        11: "IX: AMSIX"
        20: "core"
        21: "core"
      listen: :161

**Comment by vincentbernat** [comment, marked as answer]
Did you restart both the orchestrator and the demo containers?

**Comment by kronicklez** [comment]
Hi vincent,

Problem solved. Look like i need to change all info need before execute command "docker compose up -d". By the way may i know whether in the demo given can get graph like i attach? I cannot get graph like attach in demo.

![akvorado-graph](https://github.com/user-attachments/assets/9392685c-d682-4870-bf85-0ccd5ef788b9)

Thanks appreciate your feedback

**Reply by kronicklez** [reply]
Hi Vincent,

Just to update i already get that graph. Sorry my mistake.

Thanks
